packages/migrate/src/commands/MigrateStatus.ts

```diff
@@ -165,16 +162,23 @@ ${diagnoseResult.history.unpersistedMigrationNames.join('\n')}`
       //                 - Suggest calling `prisma migrate resolve --applied <migration-name>`
 
       if (listMigrationDirectoriesResult.migrations.length === 0) {
-        return new HowToBaselineError().message
+        console.error(`The current database is not managed by Prisma Migrate.
+        
+Read more about how to baseline an existing production database:
+${link('https://pris.ly/d/migrate-baseline')}`)
+        // Exit 1 to signal that the status is not in sync
+        process.exit(1)
```

Comment:
If we can't find a migration table in the db & no existing migration files -> exit 1